blockchain/tx_list_test.go

```diff
@@ -53,3 +57,147 @@ func TestStrictTxListAdd(t *testing.T) {
 		}
 	}
 }
+
+// TestTxListReadyWithGasPriceALL check whether ReadyWithGasPrice() works well.
```

Comment:
which function is correct?

* TestTxListReadyWithGasPriceALL
* TestTxListReadyWithGasPrice

Reply:
`TestTxListReadyWithGasPrice` is correct. I modified mistypo.